### PR TITLE
Fix prometheus exporter when tags are missing.

### PR DESCRIPTION
**Pull request by bogdandrutu**
Fixes https://github.com/census-instrumentation/opencensus-go/issues/659

### CODE REVIEW

File: exporter/prometheus/prometheus.go [hunk 1]
```diff
@@ -255,14 +255,6 @@ func tagKeysToLabels(keys []tag.Key) (labels []string) {
 	return labels
 }
 
-func tagsToLabels(tags []tag.Tag) []string {
-	var names []string
-	for _, tag := range tags {
-		names = append(names, internal.Sanitize(tag.Key.Name()))
```
Comment (odeke-em) [inline exporter/prometheus/prometheus.go]: Why did you remove this? It is imperative that we sanitize keys for Proemtheus otherwise "org/foo" will fail for Prometheus which expects "org_foo".

Comment (bogdandrutu) [reply exporter/prometheus/prometheus.go]: This is unused as you can see I did not change any code that calls it, and this seems to be a package private function.

Comment (odeke-em) [reply exporter/prometheus/prometheus.go]: Gotcha, but it is noisy in this PR and unrelated to this change.

Comment (bogdandrutu) [reply exporter/prometheus/prometheus.go]: I see your point, but dead code is the thing that I "like" the most :) happy to do a different PR if you feel strong.

Comment (bogdandrutu) [reply exporter/prometheus/prometheus.go]: Reverted and I would send a different PR.
